data/org.freedesktop.portal.Usb.xml

```diff
@@ -100,6 +100,7 @@
     <method name="EnumerateDevices">
       <arg type="a{sv}" name="options" direction="in"/>
       <arg type="a(sa{sv})" name="devices" direction="out"/>
+      <annotation name="org.qtproject.QtDBus.QtTypeName.Out0" value="QList&lt;QPair&lt;QString,QVariantMap&gt;&gt;"/>
```

Comment:
We put the annotations over the arguments they apply to.

Reply:
done